plugins/github/server/processors/GitHubProcessor.ts

```diff
@@ -0,0 +1,73 @@
+import { App, Octokit } from "octokit";
+import { IntegrationService, IntegrationType } from "@shared/types";
+import env from "@server/env";
+import { Integration, IntegrationAuthentication } from "@server/models";
+import BaseProcessor from "@server/queues/processors/BaseProcessor";
+import { sequelize } from "@server/storage/database";
+import { IntegrationEvent, Event } from "@server/types";
+
+export default class GitHubProcessor extends BaseProcessor {
```

Comment:
```suggestion
export default class IntegrationDeletedProcessor extends BaseProcessor {
```

Can we make this more generic, catch github as a special case and then _any other_ integration that gets deleted will be hard deleted here too. I don't want to be in a situation where we have to remember to create a processor for all other integration types